### DISCUSSION TITLE
Web screen goes black and glitches

### DISCUSSION THREAD

**Discussion by chienprovjp** [rocksdanister/lively#1807] [Q&A]
Hello, im just using the app recently and i cant find a solution to this problem. The screen glitches and sometimes it goes black completely when i use web app like chrome edge... but it is totally normal with the others. I notice that it only happens when i use web app and i open it fullscreen, i dont know why. Can anyone help me with this ;-; ??

**Comment by rocksdanister** [comment]
1. From where did you download Lively?
2. What is the version number of Lively (check be opening About page.)

**Comment by chienprovjp** [comment]
I download it from microsoft store and the version number is "v2.0.7.0 store"
